src/check_result.jl

```diff
@@ -24,6 +24,13 @@ function test_approx(
     @test_msg msg isapprox(actual, expected; kwargs...)
 end
 
+# Comparing eigen factoriztions is tricky, because the order and normalization of eigen 
+# vectors is not unique. Therefore we just calculate the original matrix and compare that
+# instead.
+function test_approx(actual::Eigen, expected::Eigen, msg=""; kwargs...)
```

Comment:
I wonder if we should do this for all subtypes of `Factorization` ?

Reply:
Hm, yes, maybe? Although I am not really familiar with those, so perhaps we should wait until an actual use case comes up?

Reply:
Fair enough.